docs/reuse/apps/app-update.md

```diff
@@ -12,3 +12,7 @@ To update the app, do the following:
         1. Click **Next**. You will be redirected to the **Preview & Done** section.
 
 Your upgraded app will be installed in the **Installed Apps** folder and dashboard panels will start to fill automatically.
+
+:::note
+Go to the **Release Notes** tab to see the change log for new updates in the app.
```

Comment:
```suggestion
For individual app updates, see [**Service Release Notes**](/release-notes-service).
```

Reply:
@kimsauce, the **Release Notes** tab appears in the App Catalog for apps that have an upgrade available:

<img width="986" alt="Screenshot 2024-06-11 at 4 01 17 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/127961815/e62d9752-61de-4236-a88b-60bc8aa7d6c2">

Reply:
Oh ok, thank you for clarifying! I don't think I ever noticed that tab was there.